index.js

```diff
@@ -23,17 +23,19 @@ function throwError (customLogger, message, error) {
   }
 }
 
-function generateCommands (filePaths, customLogger) {
+function generateCommands(filePaths, customLogger) {
   const commands = [];
 
   for (let filePath of filePaths) {
     const normalizedFile = normalizeFile(filePath, customLogger);
     if (normalizedFile) {
+      // Escape single quotes in the file path
```

Comment:
> I have updated the generateCommands function to escape single quotes in file paths. This is accomplished by replacing each single quote in the file path with two single quotes (''), which is the standard method for escaping single quotes in PowerShell.

What you have explained here is the important part. Explain that in the comment in a clear/concise way.